arbitrum-docs/bold/partials/_bold-public-preview-banner-partial.md

```diff
@@ -0,0 +1,9 @@
+:::caution ALPHA RELEASE, PUBLIC PREVIEW DOCS
+
+The BOLD dispute protocol is currently deployed on a public testnet & is tagged as an `alpha` release. The code has not been audited, and should **not be used in production scenarios**. This documentation is currently in [public preview](../concepts/public-preview-expectations.md).
```

Comment:
```suggestion
The BOLD dispute protocol is currently deployed on a public testnet and is tagged as an `alpha` release. The code has not been audited, and should **not be used in production scenarios**. This documentation is currently in [public preview](../concepts/public-preview-expectations.md).
```

Following the style conventions [here, point 10](https://docs.arbitrum.io/for-devs/contribute#style-conventions), we should limit the use of characters like '&' and '/'.